src/main/php/org/bovigo/vfs/vfsStream.php

```diff
@@ -1,58 +1,80 @@
 <?php
+
 declare(strict_types=1);
+
 /**
  * This file is part of vfsStream.
  *
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
- *
- * @package  org\bovigo\vfs
  */
+
 namespace org\bovigo\vfs;
-use org\bovigo\vfs\content\LargeFileContent;
+
+use DirectoryIterator;
+use InvalidArgumentException;
 use org\bovigo\vfs\content\FileContent;
+use org\bovigo\vfs\content\LargeFileContent;
 use org\bovigo\vfs\visitor\vfsStreamVisitor;
+use function array_map;
+use function explode;
+use function file_get_contents;
+use function filetype;
+use function function_exists;
+use function implode;
+use function is_array;
+use function is_string;
+use function octdec;
+use function posix_getgid;
+use function posix_getuid;
+use function preg_match;
+use function rawurldecode;
+use function sprintf;
+use function str_replace;
+use function strlen;
+use function strpos;
+use function substr;
+use function trim;
```

Comment:
it wasn't a manual fix. it is forced by doctrine
But does it harm to have them? it does harm to have them not. or we have to force all functions to be fully qualified names.

Reply:
It definitely doesn't harm having them. I think it's all personal preference. Hopefully someone else comments if they have a preference in either direction. 

I know my preference is probably because of the Symfony standard. I've never used the Doctrine standard for any projects before.